style/non-code/README.md

```diff
@@ -0,0 +1,19 @@
+# Non-code
+
+A spot to record non-code things, e.g. email signatures!
+
+## Email Signature
+
+```
+Jeremy Loyd
+Sparkbox, Creative Director
+123 Webster Street, Studio 2
+Dayton, OH 45402
+937.401.0915
+www.seesparkbox.com
+```
+
+- First line - `Arial, Bold, 16px, #33717a`
```

Comment:
Wonder if those ought to be updated to Avenir Next, since that is a brand font…and it is on Macs.